PagarMe.ConsoleTest/Program.cs

```diff
@@ -20,7 +21,7 @@ private static void Main(string[] args)
             {
                 AccountManagementKey = "amk",
                 MpToken = "token",
-                SecretKey = "sk",
+                SecretKey = "sk_test_zL1YeWiWqfrx5GgZ",
```

Comment:
Não deixar a sua sk hardcoded, manter apenas sk como forma de mostrar pra quem for usar essa classe que deve ser usado uma sk de teste.

Reply:
Acabei deixando passar. Removido.